README.md

```diff
@@ -51,6 +52,21 @@ After you successfully set up the project, you will have to go through the follo
    * use it as the return value of the method `getBotToken()` inside `StickerifyBot`
 3. Run the `Main` class to start the bot, it will be now able to answer messages in Telegram
 
+
+## Launch The Bot Using Docker
+Prerequisites
+
+Make sure you have already installed both Docker Engine and Docker Compose. You don’t need to install other programs like Python  as they are provided by Docker images.
+
+1. You can see how to start with docker compose here.
+2. To run the bot locally  with Docker, we need to set Passing Environment Variables Into the Dockerfile you created in 1.
+3. To do that, When we want to set this argument, we pass it with the –build-arg flag:
+
+docker build -t build-name --build-arg name=arg-name.
```

Comment:
To improve its readability I suggest to put this command inside a markdown text with shell syntax, if you are not familiar with them they are written as follows:
````markdown
   ```shell
   docker build -t build-name --build-arg name=arg-name .
   ```
````

Maybe we can also try to be more precise with our command, for instance we can specify that in our case the actual command is 
````shell
docker build -t stickerify --build-arg STICKERIFY_TOKEN={{TOKEN}} .
````
where `{{TOKEN}}` is the token retrieved from [BotFather](https://t.me/BotFather)

Reply:
Readability has been improved upon